runtime/onert/core/include/backend/IConfig.h

```diff
@@ -42,6 +42,8 @@ struct IConfig
   virtual bool supportDynamicTensor() = 0;
   virtual bool supportFP16() = 0;
 
+  virtual void sync() const {}
```

Comment:
Having this function in `sync` could be weird, but it seems like there is no better place.